ActivityLauncherApp/src/main/res/values/themes.xml

```diff
@@ -1,6 +1,6 @@
-<?xml version="1.0" encoding="utf-8"?>
-<resources>
-    <style name="AppTheme" parent="Theme.AppCompat.DayNight">
-        <item name="dialogCornerRadius">24dp</item>
-    </style>
-</resources>

```

Comment:
I did a CRLF to LF conversion here as it was displayed incorrectly in diff and hopefully it will be consistent from now on.